src/functions/sendCandidateResults/application/templates/next-steps.ts

```diff
@@ -0,0 +1,260 @@
+- a way of teaching pupils the skills, knowledge and understanding 
+
+^[Find out more about the national standards and syllabus](https://www.gov.uk/government/collections/national-driving-and-riding-standards?utm_source=dvsa&utm_medium=email&utm_campaign=adi-part-2-test&utm_content=pass). 
+
+## 3. Get practice teaching
+
+You have the option to apply for a 6 month trainee licence to get experience teaching pupils.
+
+You must have had at least 40 hours of training from a qualified ADI in providing driving instruction (at least 10 of which were done in a car) to apply.
+
+^[Find out more about getting a trainee driving instructor licence](https://www.gov.uk/trainee-driving-instructor-licence-the-rules?utm_source=dvsa&utm_medium=email&utm_campaign=adi-part-2-test&utm_content=pass). 
+
+## 4. Book your ADI part 3 test
+
+When you and your trainer agree you're ready, book your ADI part 3 test.
+
+You must book the ADI part 3 test within 2 years of the date you passed the ADI part 1 (theory) test. If you do not, you'll have to start the application process again.
+
+^[Book your ADI part 3 test](https://www.gov.uk/book-driving-test?utm_source=dvsa&utm_medium=email&utm_campaign=adi-part-2-test&utm_content=pass).
+`;
+
+export const nextStepsAdi3FirstOrSecondFailTemplate = `
+# What to do next
+
+## 1. Get a registered trainer
+
+Get a registered trainer to help you prepare for your next attempt if you do not already have one.
+
+^[Find driving instructor training courses](https://www.gov.uk/find-driving-instructor-training?utm_source=dvsa&utm_medium=email&utm_campaign=adi-part-3-test&utm_content=unsuccesful-1st-2nd). 
+
+## 2. Study the national standards
+
+To help you prepare, you should study:
+
+- the national standard for driver and rider training
+- the national standard for driving cars and light vans
+- the learning to drive a car or light van syllabus
+
+These documents set out:
+
+- the skills, knowledge and understanding you will need to pass the ADI part 3 test
+- the skills, knowledge and understanding you will help your pupils to learn
+- a way of teaching pupils the skills, knowledge and understanding 
+
+^[Find out more about the national standards and syllabus](https://www.gov.uk/government/collections/national-driving-and-riding-standards?utm_source=dvsa&utm_medium=email&utm_campaign=adi-part-3-test&utm_content=unsuccessful-1st-2nd). 
+
+## 3. If you have a trainee driving instructor licence
+
+If you chose the ‘extra training’ option (option 2) when you applied for your trainee licence, you must do 5 hours of extra training before you take the test again.
+
+^[Find out more about what you need to do if you have a trainee driving instructor licence](https://www.gov.uk/trainee-driving-instructor-licence-the-rules/options-when-you-apply-for-a-trainee-licence?utm_source=dvsa&utm_medium=email&utm_campaign=adi-part-3-test&utm_content=unsuccessful-1st-2nd). 
+
+---
```

Comment:
whats this?

Reply:
They are intentional but think this is another point of contention regarding inconsistencies between the different templates. Will make sure to raise it when the templates actually get reviewed